views/searchjob.handlebars

```diff
@@ -52,7 +52,7 @@
             </div>
                 <div class="row">
                   <div class="col-md-6">
-                    <a id="submitsearch" class="btn btn-lg index-button btn-block" href="/login" role="button">Submit</a>
+                    <a id="submitsearch" class="btn btn-lg index-button btn-block" href="resumepdf.handlebars" role="button">Submit</a>
```

Comment:
Links should not display ".handlebars". The route needs to include "/". We are providing routes, not links to files.